mmyolo/deploy/models/dense_heads/yolov6_head.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+from functools import partial
+from typing import List, Optional, Tuple
+
+import torch
+from mmdeploy.codebase.mmdet import get_post_processing_params
+from mmdeploy.codebase.mmdet.models.layers import multiclass_nms
+from mmdeploy.core import FUNCTION_REWRITER
+from mmengine.config import ConfigDict
+from mmengine.structures import InstanceData
+from torch import Tensor
+
+from mmyolo.deploy.models.layers import efficient_nms
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='mmyolo.models.dense_heads.yolov6_head.'
+    'YOLOv6Head.predict_by_feat')
+def yolov6_head__predict_by_feat(ctx,
```

Comment:
```suggestion
def yolov6_head_predict_by_feat(ctx,
```

Reply:
MMDeploy modifies the rewritten function name to this format.
https://github.com/open-mmlab/mmdeploy/blob/dev-1.x/mmdeploy/codebase/mmdet/models/dense_heads/yolox_head.py#L18